Makefile

```diff
@@ -14,7 +14,7 @@
 
 BUNDLE_VERSION = 4.1.11
 BUNDLE_EXTENSION = crcbundle
-CRC_VERSION = 1.0.0-beta.2
+CRC_VERSION = 1.0.0-beta3
```

Comment:
why the change from beta.2 to beta3? I'd keep the same format for now

Reply:
Ah you mean `beta.3` ?